lib/mix/tasks/validate_company_file.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Mix.Tasks.ValidateCompanyFile do
+  use Mix.Task
+
+  alias Companies.Industries
+  alias Companies.Schema.Company
+
+  @moduledoc """
+  Validates a filepath to a company.
+
+      $ mix validate_company_file priv/companies/abc.exs
+
+  This is to help with data consistency.
+  """
+
+  @impl Mix.Task
+  @doc false
+  def run([file_path]) do
```

Comment:
It might be worth doing a `Path.wildcard` so the function can run over all the files. It would be simpler to do it in Elixir than doing it via a bash shell, and it would be easier for devs to run locally.

Reply:
I was hesitant to run it over all the files as a good majority of the existing company will likely need to be updated. My thought was adding this so that new or updates to company files could have some data consistency for industries.

This was my attempt at something similar to an ecto changeset validation rule without having or adding ecto back to the project. 

I'm open to other ideas or approaches.